doc/api/assert.md

```diff
@@ -2539,6 +2539,96 @@ assert.throws(throwingFirst, /Second$/);
 // AssertionError [ERR_ASSERTION]
 ```
 
+## `assert.matchObject(actual, expected[, message])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `actual` {any}
+* `expected` {any}
+* `message` {string|Error}
+
+Tests equivalence between the actual and expected parameters by performing
+a deep comparison, ensuring that all properties and their values are equal,
+allowing type coercion where necessary.
+
+**Strict assertion mode**
+
+An alias of \[`assert.matchObjectStrict()`]\[].
```

Comment:
```suggestion
An alias of [`assert.matchObjectStrict()`][].
```

Reply:
@aduh95 running the `make format-md` will revert it back to `An alias of \[`assert.matchObjectStrict()`]\[].`

Reply:
Yeah, you need to define the reference at the end of the file – or to remove the link